src/view/reviewManager.ts

```diff
@@ -1025,6 +1028,55 @@ export class ReviewManager implements vscode.DecorationProvider {
 		await this.validateState();
 	}
 
+	public async createPullRequest(): Promise<void> {
+		const branchName = this._repository.state.HEAD.name;
+		let potentialTargetRemotes = this._prManager.getGitHubRemotes();
```

Comment:
We could call `getPullRequestDefaults` here to get the default values for the quickpicks. That'll handle guessing reasonable initial values.